integrated_channels/api/v1/logs/views.py

```diff
@@ -56,6 +56,7 @@ class LearnerSyncStatusViewSet(PermissionRequiredForIntegratedChannelMixin, view
     """
     Sync-status APIs for `LearnerDataTransmissionAudit` implementation items.
     """
+    # serializer_class = LearnerSyncStatusSerializer
```

Comment:
Is this an informational comment or a piece of code left in?

Reply:
it's been removed 😄